evm-ds/src/precompiles/modexp.rs

```diff
@@ -0,0 +1,154 @@
+use evm::executor::stack::{PrecompileFailure, PrecompileOutput};
+use evm::{Context, ExitError, ExitSucceed};
+use num_bigint::BigUint;
+use num_integer::Integer;
+use primitive_types::U256;
+use std::borrow::Cow;
+
+pub(crate) fn modexp(
```

Comment:
Could you please add a comment what was borrowed from other code, and what is your authorship.  (I know I haven't done it for ecrecover, need to do too ...)